<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandsSettingsWidget</class>
 <widget class="QWidget" name="CommandsSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commands Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="FilterLineEdit" name="filterEdit"/>
   </item>
   <item>
    <widget class="QTreeView" name="view">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_3">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetAllButton">
       <property name="toolTip">
        <string>Reset all to default</string>
       </property>
       <property name="text">
        <string>Reset All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">Widgets/FilterLineEdit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
